securitas/utility/messaging.py

```diff
@@ -0,0 +1,26 @@
+import sys
+import traceback
+
+from fedora_messaging import api, exceptions as fml_exceptions
+import backoff
+
+from securitas import app
```

Comment:
I would recommend using `from flask import current_app` instead, it can help solve circular imports.